AdminWebsite/AdminWebsite/Services/UserAccountService.cs

```diff
@@ -280,6 +281,9 @@ public async Task AssignParticipantToGroup(string username, string userRole)
                 case JohRole:
```

Comment:
Is it worth checking the two parameters coming in and also a default for the case block?

Reply:
We have unit test to check the userName that throws exceptions if this is null> Should_throw_exception_when_add_user_to_group_has_null_user_name()

The userRole can be anything and it does not need to meet the case block ( no default needed )